modules/libusb/1.0.28/overlay/BUILD.bazel

```diff
@@ -0,0 +1,95 @@
+load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
+
+package(default_visibility = ["//visibility:public"])
```

Comment:
For modules that don't have Bazel support upstream, I would prefer more restrictive visibility since we may need to change the structure over time. Would it work to only make `:usb` and `:libusb` visible for now?

Reply:
fixed at https://github.com/bazelbuild/bazel-central-registry/pull/4112/commits/c35ee5cd305436d8b72152343523197ee0d853c5